frontend/src/components/channels/ChannelsCards.tsx

```diff
@@ -80,55 +79,51 @@ export function ChannelsCards({
                         <div className="flex-1 whitespace-nowrap text-ellipsis overflow-hidden">
                           {alias}
                         </div>
-                        <DropdownMenu>
-                          <DropdownMenuTrigger asChild>
-                            <Button size="icon" variant="ghost">
-                              <MoreHorizontal className="h-4 w-4" />
-                            </Button>
-                          </DropdownMenuTrigger>
-                          <DropdownMenuContent align="end">
-                            <DropdownMenuItem className="flex flex-row items-center gap-2 cursor-pointer">
-                              <ExternalLink
-                                to={`https://mempool.space/tx/${channel.fundingTxId}`}
-                                className="w-full flex flex-row items-center gap-2"
-                              >
-                                <ExternalLinkIcon className="w-4 h-4" />
-                                <p>View Funding Transaction</p>
-                              </ExternalLink>
-                            </DropdownMenuItem>
-                            <DropdownMenuItem className="flex flex-row items-center gap-2 cursor-pointer">
-                              <ExternalLink
-                                to={`https://amboss.space/node/${channel.remotePubkey}`}
-                                className="w-full flex flex-row items-center gap-2"
-                              >
-                                <ExternalLinkIcon className="w-4 h-4" />
-                                <p>View Node on amboss.space</p>
-                              </ExternalLink>
-                            </DropdownMenuItem>
-                            {channel.public && (
-                              <DropdownMenuItem
-                                className="flex flex-row items-center gap-2 cursor-pointer"
-                                onClick={() => editChannel(channel)}
-                              >
-                                <HandCoins className="h-4 w-4" />
-                                Set Routing Fee
+                        <AlertDialog>
+                          <DropdownMenu>
```

Comment:
Can you move this dropdown menu to a separate component in this folder? I realized its the same for both the channels table and cards

Reply:
Done!